app/src/main/java/agency/five/codebase/android/movieapp/ui/movieDetails/MovieDetailsScreen.kt

```diff
@@ -0,0 +1,257 @@
+) {
+    Box(
+        modifier = modifier
+            .fillMaxWidth()
+            .height(dimensionResource(id = R.dimen.movie_details_hero_height))
+    ) {
+        AsyncImage(
+            model = movieDetailsViewState.imageUrl,
+            contentDescription = stringResource(id = R.string.movie_image),
+            placeholder = painterResource(id = R.drawable.ic_launcher_background),
+            error = painterResource(id = R.drawable.ic_launcher_background),
+            contentScale = ContentScale.Crop,
+            modifier = Modifier.fillMaxSize()
+        )
+        Column(
+            modifier = Modifier.fillMaxSize(),
+            verticalArrangement = Arrangement.Bottom
+        ) {
+            Row(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(
+                        start = dimensionResource(id = R.dimen.movie_details_hero_user_score_padding)
+                    ),
+                verticalAlignment = Alignment.CenterVertically,
+
+                ) {
+                UserScoreProgressBar(
+                    userScore = movieDetailsViewState.voteAverage,
+                    modifier = Modifier.size(
+                        dimensionResource(id = R.dimen.user_score_progress_bar_size)
+                    )
+                )
+                Text(
+                    text = stringResource(id = R.string.user_score_text),
+                    style = MovieHeroSupportText,
+                    maxLines = 1,
+                    overflow = TextOverflow.Ellipsis,
+                    modifier = Modifier.padding(start = MaterialTheme.spacing.small)
+                )
+            }
+            Spacer(modifier = Modifier.height(MaterialTheme.spacing.small))
+            Text(
+                text = movieDetailsViewState.title,
+                style = MovieHeroTitle,
+                maxLines = 1,
+                overflow = TextOverflow.Ellipsis,
+                modifier = Modifier.padding(
+                    start = dimensionResource(id = R.dimen.movie_details_hero_movie_title_padding)
+                )
+            )
+            Spacer(modifier = Modifier.height(MaterialTheme.spacing.small))
+            FavoriteButton(
+                modifier = Modifier
+                    .padding(
+                        start = dimensionResource(id = R.dimen.movie_details_hero_movie_title_padding),
+                        bottom = MaterialTheme.spacing.medium
+                    ),
+                isFavorite = movieDetailsViewState.isFavorite,
+                onCLick = {}
+            )
+        }
+    }
+}
+
+@Composable
+fun MovieDetailsOverviewSection(
+    movieDetailsViewState: MovieDetailsViewState,
+    modifier: Modifier = Modifier
+) {
+    Column(
+        modifier = modifier
+            .fillMaxWidth()
+            .padding(
+                start = MaterialTheme.spacing.medium,
+                end = MaterialTheme.spacing.medium
```

Comment:
Since padding is set here, `lazyHorizontalGrid` won't be scrollable to the end of the screen.
You will have to set padding on each element, and on `LazyHorizontalGrid` you need to set `contentPadding`. You have the sam problem on `HomeScreen` with LazyRow where items are `MovieCards`, please fix it in both places.